content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -201,60 +241,79 @@ git push -u origin main
 
 You're ready to import your project into the Prisma Data Platform.
 
-## 7. Importing your Project into the Prisma Data Platform
+## 8. Import your Project into the Prisma Data Platform
 
 With Cloudflare Workers, you can't directly access your database because there is no TCP support. Fortunately, Prisma has your back with the [Data Proxy](/data-platform/data-proxy).
 
-To get started, sign up for a free [Prisma Data Platform account](https://cloud.prisma.io/).
+1. To get started, sign up for a free [Prisma Data Platform account](https://cloud.prisma.io/).
+   <Admonition type="info">
+   
+   **Note**<br /><br />
+   You only need to have a GitHub account to automatically sign up when you log in for the first time.
+   </Admonition>
 
-![Signup for Prisma Data Platform](./450-02-signup-pdp.png)
+   ![Signup for Prisma Data Platform](./450-02-signup-pdp.png)
 
-Once you're in, click **New Project**, then **Import a Project**. Fill in the repository and project details, and then click **Create Project**.
+2. Once you're in, click **New Project**, then **Import a Project**.
```

Comment:
```suggestion
2. After you are logged in to the Prisma Data Platform, click **New Project**, and then click **Import a Project**.
```

Reply:
"Once" is a word to avoid when writing about time-based events... can be confusing for non-Native English speakers, since it is also a word about numbers (once, twice, thrice, etc).

Reply:
I thought I knew better :) Changed.